### PR TITLE
Attributes and Revision id section in Policyfiles of Chef Infra Server

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
Signed-off-by: Dishank Tiwari <dtiwari@progress.com>

### :nut_and_bolt: Description: What code changed, and why?

Two new options have been added to the policyfiles section of the Chef Infra Server:
- Attributes
- Revision id

### :chains: Related Resources

Reference PRs:
- https://github.com/chef/automate/pull/5374
- https://github.com/chef/automate/pull/5377

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Review by himanshi-chhabra — APPROVED** [review]
LGTM Thanks @dishanktiwari2501 :tada:

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 1]
```diff
@@ -453,7 +453,10 @@ The Chef Infra Server integration lets you:
 
 - Search for a specific policyfiles
 - View all the policyfiles
-- Details of policyfiles
+- View the details of policyfiles:
+  - Content
+  - Attributes
+  - Revision id
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:459]: ID should also be capitalized.
```suggestion
  - Revision ID
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 2]
```diff
@@ -464,10 +467,22 @@ Use the search bar on the Policyfile tab (**Chef Infra Server > Policyfile**) to
 
 #### Details of Policyfiles
 
-Select a policyfile to view the details, that is the **Revision Id** and the **METADATA** of the policyfile. The **Content** tab contains the list of all the **Included Policies** and **Run List**.
+Select a policyfile to view the details of the policyfile. The details are the **METADATA**, **Content**, **Attributes** and **Revision id** of the policyfiles.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:470]:
```suggestion
Select a Policyfile to view the **METADATA**, **Content**, **Attributes**, and **Revision ID** of that Policyfile.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 3]
```diff
@@ -464,10 +467,22 @@ Use the search bar on the Policyfile tab (**Chef Infra Server > Policyfile**) to
...
 
 {{< figure src="/images/automate/details-of-policyfiles.png" alt="Details of Policyfiles">}}
 
+The **content** tab contains the list of all the *included policies* and *run list*, whereas the **attributes** tab displays the *default* and the *override* attributes of the policyfiles.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:474]:
```suggestion
The **Content** tab contains the list of all the *included policies* and *run list*, whereas the **Attributes** tab displays the *default* and the *override* attributes of a Policyfile.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 4]
```diff
@@ -464,10 +467,22 @@ Use the search bar on the Policyfile tab (**Chef Infra Server > Policyfile**) to
...
+
+{{< figure src="/images/automate/attributes-tab-of-policyfiles.png" alt="Attributes tab of Policyfiles">}}
+
+You can **Expand** and **Collapse** the *default* and *override* attributes by selecting the option.
+
+{{< note >}}
+You can find the **revision id** window directly by selecting the ellipses icon of a policyfile. Selecting the option opens a pop-up which contains the detailed revision id.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:481]:
```suggestion
You can find the **Revision ID** window directly by selecting the ellipses icon of a Policyfile. Selecting the option opens a pop-up which contains the detailed revision ID.
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 5]
```diff
@@ -453,7 +453,10 @@ The Chef Infra Server integration lets you:
 
 - Search for a specific policyfiles
 - View all the policyfiles
-- Details of policyfiles
+- View the details of policyfiles:
+  - Content
+  - Attributes
+  - Revision ID
 - Delete policyfiles
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:460]:
```suggestion
- View the details of Policyfiles:
  - Content
  - Attributes
  - Revision ID
- Delete Policyfiles
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 6]
```diff
@@ -453,7 +453,10 @@ The Chef Infra Server integration lets you:
 
 - Search for a specific policyfiles
 - View all the policyfiles
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:455]:
```suggestion
- Search for a specific Policyfiles
- View all the Policyfiles
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 7]
```diff
@@ -464,10 +467,22 @@ Use the search bar on the Policyfile tab (**Chef Infra Server > Policyfile**) to
...
+
+{{< note >}}
+You can find the **Revision ID** window directly by selecting the ellipses icon of a Policyfile. Selecting the option opens a pop-up which contains the detailed revision ID.
+{{< /note >}}
+
+{{< figure src="/images/automate/revision-id-of-policyfiles.png" alt="Revision id of Policyfiles">}}
+
 #### Delete Policyfiles
 
 Delete individual policyfiles by selecting the ellipses icon {{< fontawesome class="fas fa-ellipsis-h" >}} and then **Delete**:
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/chef_infra_server.md:488]:
```suggestion
Delete individual Policyfiles by selecting the ellipses icon {{< fontawesome class="fas fa-ellipsis-h" >}} and then **Delete**:
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**